### DISCUSSION TITLE
Additional command to pull device list?

### DISCUSSION THREAD

**Discussion by town3r** [nVuln/homebridge-lg-thinq#6] [Q&A] [answered]
Is there a command I can run to pull a list of current devices similar to `wideq ls`?

**Comment by nVuln** [comment, marked as answer]
this is plugin of homebridge, so we don't provide CLI command feature
all you need is turn on homebridge debug mode and check debug logging, you can see something like this
```
[LGThinQ] Found device:  xxx-xxx-xxx-xxx-xxx: Washer (WASHER xxxxxxxx)
```
if device currently not supported, maybe
```
[LGThinQ] Device not supported:  xxx-xxx-xxx-xxx-xxx: Dryer (DRYER xxxxxxxx)
[LGThinQ] device data: *********************
```
